code/game/objects/structures/droppod.dm

```diff
@@ -112,19 +114,25 @@
 /obj/structure/droppod/proc/checklanding(mob/user)
 	var/turf/target = locate(target_x, target_y, 2)
 	if(target.density)
-		to_chat(user, "<span class='warning'>Dense Landing zone detected. Invalid area.</span>")
+		to_chat(user, "<span class='warning'>Dense landing zone detected. Invalid area.</span>")
 		return FALSE
-	if(isspaceturf(target))
+	if(isspaceturf(target)) // Just a failsafe.
 		to_chat(user, "<span class='warning'>Location outside mission parameters. Invalid area.</span>")
 		return FALSE
```

Comment:
Is this still needed?

Reply:
Again, it is needed as a failsafe if it somehow fails.

Reply:
But it wont, it has the flag?